### ISSUE TITLE
Introduction task Crowd Sourcing 

### ISSUE THREAD

**Issue by Abuteslimah99** [Meeds-io/meeds#1818]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to do a certain task over and over again bcos it's not particularly my field of expertise 

**Describe the solution you'd like**
I'd like to suggest task crowd Sourcing where I can take a task and delegate it for another contributor or even a non contributor that does not have access to that same task or the builders hub generally, to complete and I can allocate point to the contributor or guests based on initial agreement for example not everyone can curate a video or do a linkedin post or even carry out task reviews and reporting of bugs or security vulnerabilities 

**Describe alternatives you've considered**
A live chat session where contributors can work as a team 

**Additional context**
This feature will make it so much easier to carry out task efficiently as I can now concentrate on areas and tasks that I'm good in while still earning points in other tasks that I'm not so good at

**Comment by srenault-meeds** [comment]
Hi @Abuteslimah99 I am trying to understand your point but actually, it seems to go against the fact that individuals can achieve tasks depending on their ability / skill, etc.
Are you proposing something like a delegation process so you can delegate the task achievement to someone else and then get the point for yourself?

**Comment by Abuteslimah99** [comment]
Yes a delegation option but if it goes against the rules it could be more like a collaboration...where for example even though I am not authorized to review articles an authorized reviewer can delegate some reviews to me after showing interest after which he can share his points with me based on initial agreements...I sincerely apologize for replying late
